src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/Streaming/types.ts

```diff
@@ -0,0 +1,33 @@
+import { AnyToken } from '~data/index';
+
+export interface FundingSource {
+  team: string;
+  rate: {
+    amount?: number;
+    token?: string;
+    time?: string;
+    limit?: number;
+    id: string;
+  }[];
+  isExpanded: boolean;
+  id: string;
+}
+
+export interface Streaming {
+  fundingSource: FundingSource[];
```

Comment:
Similarly to my comment in #3877, I think pluralising this would give a better indication that it's an array